packages/autodoc/init.lua

```diff
@@ -371,6 +371,11 @@ function package:registerCommands ()
 
    -- Homogenizing the appearance of blocks of code
    self:registerCommand("autodoc:codeblock", function (_, content)
+      -- The way we use this command in the SILE manual is both on it's own *and* nested inside
+      -- a \raw environment. For this purpose we don't care about and don't want to bother with
+      -- making sure the leading and trailing whitespace is minimized in the SIL source, so
+      -- we trim it off here.
+      content = SU.ast.trimSubContent(content)
```

Comment:
Does this change sound right to you @Omikhleia? The whitespace being passed in the `\raw` blocks now being (legitimately) treated as actual lines seems to be the main problem that blew up the manual. My previous fix was obviously wrong, this seems right now. I did tweak the alignment of the rule just a little bit and used raise instead of manually pushing VGlue because sorting out when to add spacing or not was much easier that way. Hence the final test expectations did change just a small shim. I also did *not* place `\novbreak` before and after the cod blocks as that seems to introduce yet another problem we'd have to cancel out.

Reply:
As far as I can say viewing the code (not tested yet), it seems ok.
What is the problem your are having with novbreak, if I may ask? I've some regression (it seems) on them too, though I've not been able yet to track its origin. (Something changed in paragraphing, possibly unrelated however.)